storage/innobase/include/buf0buf.h

```diff
@@ -1543,8 +1543,7 @@ class buf_pool_t
   bool running_out() const
   {
     return !recv_recovery_is_on() &&
-      UNIV_UNLIKELY(UT_LIST_GET_LEN(free) + UT_LIST_GET_LEN(LRU) <
-                    std::min(curr_size, old_size) / 4);
+      UT_LIST_GET_LEN(free) + UT_LIST_GET_LEN(LRU) < (n_chunks_new / 4 * chunks->size);
   }
```

Comment:
We might want `TPOOL_SUPRESS_TSAN` here to avoid complaints about not holding `buf_pool.mutex`. It is not only about the `n_chunks_new` but also the old fields.
Please split the last line to avoid it exceeding 79 columns. The parentheses after `<` should not be necessary.